docs/modules/operation/pages/topology/enlinkd/introduction.adoc

```diff
@@ -22,6 +22,34 @@ The following sections describe the Enlinkd daemon and its configuration.
 Additionally, the supported link discovery implementations are described as well as a list of the SNMP MIBs that the SNMP agents must expose for EnLinkd to gather links between nodes.
 You can view detailed information about a node's connections (discovered links) and supporting link data on the node detail page within the {page-component-title} Web UI.
 
+[[ga-enlinkd-layers]]
+== Topology Layers
+
+Enhanced Linkd Topology Map supports Layers protocol maps.
+
+Click the Layers Button on the right panel of the Topology Map to access available Layers.
+
+Supported Layers Maps:
+
+*All* _(Selected by default)_ displays the topology information discovered by the Enhanced Linkd,
+for all supported protocols OSPF, ISIS, Bridge, LLDP and CDP including User Defined to generate the overall topology
+
+*CDP* displays the Cisco Discovery Protocol topology information discovered by the Enhanced Linkd
```

Comment:
```suggestion
* *CDP* – displays the Cisco Discovery Protocol topology information Enhanced Linkd discovers.
```

Reply:
Out of scope for this PR, but it would be nice if the labels in the UI were displayed correctly ... CDP instead of Cdp, UDP instead of Udp.

Reply:
Note also that UDP is missing from this list. Please add.